### PR TITLE
planificator with no producer

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: shell/app-shell/elements/arc-planner.js [hunk 1]
```diff
@@ -63,9 +63,10 @@ class ArcPlanner extends Xen.Debug(Xen.Base, log) {
   }
   async _createPlanificator(arc, userid) {
     const planificatorParam = ArcsUtils.getUrlParam('planificator');
+    const onlyConsumer = ArcsUtils.getUrlParam('onlyConsumer') || false;
```
Comment (lindner) [inline shell/app-shell/elements/arc-planner.js]: maybe switch to 
 ArcsUtils.getUrlParam('onlyConsumer') === 'true' to force a boolean value?

Comment (mmandlis) [reply shell/app-shell/elements/arc-planner.js]: done

**Comment by mmandlis** [comment]
yes, we should document them, but I'd rather wait just a tiny bit until all the refactoring and wiring up is finished, wdyt?

**Comment by mmandlis** [comment]
Thanks for filing the issue!

**Pull request merged**